dictionary.txt

```diff
@@ -235,3 +235,123 @@ webkit-sticky
 webkit-text-fill-color
 whatsApp
 xmlns
+isHeight100Vh
+mobilePadding
+.swiper-slide-thumb-active
+ReactComponent
+RightArrorCircle
+StyledRightArrorCircle
```

Comment:
StyledRightArrorCircle -> StyledRightArrowCircle

Reply:
I'll correct it.